src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ReviewChangeStartYearRequest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sfa.Tl.ResultsAndCertification.Models.Contracts.AdminDashboard
+{
+    public class ReviewChangeStartYearRequest
+    {
+        public int PathwayId { get; set; }        
```

Comment:
I think this should be called **RegistrationPathwayId**. That would make clear it is the Id from the table TqRegistrationPathway, rather the from TlPathway table.

Reply:
updated the property name